test/dhtest.c

```diff
@@ -580,19 +491,23 @@ static int run_rfc5114_tests(void)
     BN_free(priv_key);
     OPENSSL_free(Z1);
     OPENSSL_free(Z2);
-
-    fprintf(stderr, "Initialisation error RFC5114 set %d\n", i + 1);
-    ERR_print_errors_fp(stderr);
+    TEST_error("Initialisation error RFC5114 set %d\n", i + 1);
```

Comment:
WIth most TEST_infos being immediately after TEST_xxx or TEST_error, it is unlikely anything would be printed.  Still, I understand the point.  This is one that is likely better left until after we've more experience with the framework's output.